src/main/webapp/scripts/region-selector.js

```diff
@@ -0,0 +1,17 @@
+function getRegions() {
+  return gapi.client.youtube.i18nRegions.list({
+    'part': [
+      'snippet'
+    ],
+    'hl': 'es_MX'
```

Comment:
nit: maybe add a comment to explain what this field is about?

Reply:
Agreed. Can't tell what this is either.

Reply:
I realized I didn't even need it 😅 I don't remember how it got there, probably from some documentation somewhere...